internal/operator-controller/rukpak/render/registryv1/generators/generators.go

```diff
@@ -29,6 +30,8 @@ import (
 const (
 	tlsCrtPath = "tls.crt"
 	tlsKeyPath = "tls.key"
+
+	labelKubernetesNamespaceMetadataName = "kubernetes.io/metadata.name"
```

Comment:
Is this right? Should it be `metadata.namespace`? Because this ought to be the *name* of the resource, not the namespace?

Reply:
I think so. We're selecting for the name of namespace. In more recent kube versions, the label "kubernetes.io/metadata.name" started to be stamped out to allow for label selection on the resource name. So, if targetNamespaces = ["ns1"], it will limit the webhook to namespaces with label "kubernetes.io/metadata.name" in ["ns1"] => the namespace with name ns1. Or, am I misunderstanding your question?

Reply:
I think i may have been confused, but you're looking at namespace resources for this selector, so I think we're good.